lib/ext/heroku/command/addons.rb

```diff
@@ -251,7 +256,7 @@ def attach
       case response.status
       when 201
         display("done")
-        action("Setting #{response.body["name"]} vars and restarting #{app}") do
+        action("Setting #{response.body["name"]} and restarting #{app}") do
```

Comment:
Yeah. It relies on implementation details far too much for my liking, but yes.

Ideally, if heroku/api#3728 prototype holds enough merit to pursue that direction, add-on vars won't be set on the app directly anyway (though it will still be restarted and see those vars at runtime). Making the language less release number-oriented may be necessary to keep this message meaningful if we go that direction.